tests/routerTest.php

```diff
@@ -34,28 +34,6 @@ public function testInvalidFunctionSource()
         require 'router.php';
     }
 
-    public function testInvalidFunctionTarget()
-    {
-        $this->expectException('RuntimeException');
-        $this->expectExceptionMessage('FUNCTION_TARGET is not set');
-
-        putenv('FUNCTION_SOURCE=' . __DIR__ . '/../examples/hello/index.php');
-        putenv('FUNCTION_TARGET');
-        putenv('FUNCTION_SIGNATURE_TYPE');
-        require 'router.php';
-    }
-
-    public function testInvalidFunctionSignatureType()
-    {
-        $this->expectException('RuntimeException');
-        $this->expectExceptionMessage('FUNCTION_SIGNATURE_TYPE is not set');
-
-        putenv('FUNCTION_SOURCE=' . __DIR__ . '/../examples/hello/index.php');
-        putenv('FUNCTION_TARGET=helloHttp');
-        putenv('FUNCTION_SIGNATURE_TYPE');
-        require 'router.php';
-    }
```

Comment:
It would be better to add new tests for the default behavior rather than remove these tests entirely

Reply:
Agreed. I've changed the `FUNCTION_TARGET is not set` to handle the new behavior.